### PR TITLE
Bootstrapping the self-hosted documentation for version 2.0.10

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,85 +1,81 @@
-DataStax Java Driver for Apache Cassandra
-=========================================
-
-.. image:: https://travis-ci.org/datastax/java-driver.svg?branch=2.0
-   :target: https://travis-ci.org/datastax/java-driver
-   :alt: Travis Build
-
+# Datastax Java Driver for Apache Cassandra
+
+*If you're reading this on GitHub, please note that this is the readme
```
Comment (adutra) [inline README.md]: I wonder if this paragraph is still useful. Ultimately, documentation would only be generated, and thus available, for released versions, so there is no risk of seeing documentation for an unreleased version.

File: docs.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  - title: Issues
+    href:  https://datastax-oss.atlassian.net/browse/JAVA/
+  - title: Mailing List
+    href:  https://groups.google.com/a/lists.datastax.com/forum/#!forum/java-driver-user
+  - title: IRC Channel
+    href:  irc://irc.freenode.net/datastax-drivers
+  - title: Releases
+    href:  https://github.com/datastax/java-driver/releases
+versions:
+  - name: 2.0.10
```
Comment (adutra) [inline docs.yaml]: Version 2.0.10 points to the (yet-to-be-created) tag 2.0.10. This means that the documentation can only be generated _after_ the release. Exceptionally, and only for demonstration purposes, I published the documentation for the current 2.0.10 using this PR branch, but again, in the future, the documentation should only be generated after the release.

File: driver-core/src/main/java/com/datastax/driver/core/QueryLogger.java [hunk 3]
```diff
@@ -66,7 +66,7 @@
  * The appropriate logger is chosen according to the following algorithm:
  * <ol>
  *     <li>if an exception has been thrown: use {@link #ERROR_LOGGER};</li>
- *     <li>otherwise, if the reported latency is greater than {@link #getSlowQueryLatencyThresholdMillis()} the configured threshold in milliseconds}: use {@link #SLOW_LOGGER};</li>
+ *     <li>otherwise, if the reported latency is greater than {@link #getSlowQueryLatencyThresholdMillis() the configured threshold in milliseconds}: use {@link #SLOW_LOGGER};</li>
```
Comment (adutra) [inline driver-core/src/main/java/com/datastax/driver/core/QueryLogger.java]: Small typo that I spotted while browsing the javadocs.

File: docs.yaml (hunk 2 above)
Comment (avalanche123) [inline docs.yaml]: You could use HEAD as the red until the release and change to the tag name afterwards

Comment (avalanche123) [inline docs.yaml]: s/red/ref

File: README.md (hunk 1 above)
Comment (olim7t) [inline README.md]: If you go to the project on GitHub, you'll still see the readme of the version currently in progress. That's what this comment is for.

File: driver-core/Upgrade_guide_to_2.0.rst [hunk 4]
```diff
@@ -155,13 +151,28 @@ Other API Changes
...
+   Existing implementations of this interface, once upgraded to the new method
+   signature, should continue to work as before.
+
+2. SocketOptions#getTcpNoDelay() is now TRUE by default (it was previously undefined).
+   This reflects the new behavior of Netty (which was upgraded from version 3.9.0 to
+   4.0.27): TCP_NODELAY is now turned on by default, instead of depending on the OS
+   default like in previous versions.
+
+3. Netty is not shaded anymore in the default Maven artifact. However we publish a
+   shaded artifact under a different classifier. See `Using the shaded JAR<http://datastax.github.io/java-driver/features/shaded_jar/>`_
```
Comment (adutra) [inline driver-core/Upgrade_guide_to_2.0.rst:170]: Just a note so that we do not forget to check this URL when the docs are published.

**Comment by olim7t** [comment]
Force-pushed to clean up history.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+The last release of the driver is available on Maven Central. You can install
+it in your application using the following Maven dependency:
+
+    <dependency>
+      <groupId>com.datastax.cassandra</groupId>
+      <artifactId>cassandra-driver-core</artifactId>
+      <version>2.0.10</version>
+    </dependency>
+
```
Comment (avalanche123) [inline README.md:96]: would looks better if used ```xml tag, e.g.:

``` xml
    <dependency>
      <groupId>com.datastax.cassandra</groupId>
      <artifactId>cassandra-driver-core</artifactId>
      <version>2.0.10</version>
    </dependency>
```

**Pull request merged**